organization/infrastructure.md

```diff
@@ -29,6 +47,12 @@ This shares information about what is done, how it is done, and ensures no one i
 
 We should also redeploy most of our servers regularly to make sure the install/backup scripts include everything -> even better, we should ensure that we are always able to redeploy the entire infrastructure by just having access to the `servers` repo.
 
+```{note}
+I am not versed into infra deployment, but I guess the keys and passwords to specific services / dns / cloud instances need to be stored outside the servers repo and shared between collective members.
+
+Would it then be enough to have access to server and keys repo to full redeploy?
+```
+
```

Comment:
there are tools to encrypt secrets and have them stored in git repos which are clear text (git-crypt, mozilla SOPS, ...) but let's say for the sake of argument that we have 2 repos: `servers` with the deployment code for all the infrastructure and `secrets` for the keys and access tokens, then yes:

We need to be able to re-deploy the entire infrastructure using just those 2 repos.

In other words, there is no hidden state or changes that have been applied to a server (or other part of the infrastructure) that are not contained inside the deployment code.